pages/dkp/konvoy/2.2/choose-infrastructure/aws/air-gapped/new/index.md

```diff
@@ -53,7 +53,7 @@ When you use existing infrastructure, DKP does _not_ create, modify, or delete t
     kubernetes.io/role/internal-elb = 1
```

Comment:
`kubernetes.io/cluster/CLUSTER_NAME = owned` -> `kubernetes.io/cluster/<CLUSTER_NAME> = owned` for consistency.